README.md

```diff
@@ -164,3 +164,126 @@ postman.setEnvironmentVariable("token", jsonData.token);
 4. Check if there is a cron job set to renew certificates. If there is non add the cron task below. This will try to renew the certificate twice a day every day
 
 `0 */12 * * * /root/ddw-analyst-ui/certbot.sh >/dev/null 2>&1`
+
+
+### API
+
+1.  Login and get token
+    POST - http://127.0.0.1:8000/api/auth/login/
```

Comment:
This base URL must be `https://ddw.devinit.org` ... update for every occurrence.

Reply:
Updated to production `url`